.changeset/rude-hairs-whisper.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': minor
+---
+
+When injecting routes in integrations with the `injectRoute` API, the passed route's `entryPoint` property is renamed to `entrypoint` to be consistent with other Astro APIs
```

Comment:
```suggestion
Renames the `entryPoint` property of the `injectRoute` integrations API to `entrypoint` for consistency. A warning will be shown prompting you to update your code when using the old name.
```
My prepositions may not be correct here, but I think frontloading the idea that a property has been renamed is a helpful way to go here!

Reply:
Sounds good to me 👍